src/theme/Navbar/Layout/SeqeraHeader/HeaderDesktop/NavItems/index.jsx

```diff
@@ -20,6 +22,9 @@ const NavItems = ({ isDark = false, hideMenu }) => {
     isHeaderVisible,
   } = useHeaderContext();
 
+  const location = useLocation();
+  const isOnPlatformAPI = location.pathname.includes('/platform-api');
```

Comment:
It'd be nice to scatter explanation comments and `TODO` messages all over this code, if we're expecting to revert it in the near-ish future.. (just so that we remember).


```suggestion
  // TODO: Revert this workaround once we have docs in a single build
  const location = useLocation();
  const isOnPlatformAPI = location.pathname.includes('/platform-api');
```